accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -290,7 +289,6 @@ public boolean isPreviewing() {
    */
   public void stopPreview() {
     if (mPublisher != null && isPreviewing) {
-      mPublisher.destroy();
       dettachPublisherView();
```

Comment:
You should call `mPublisher.onStop()` if you want the camera capturer to be stopped. If you don't do it and try to open the camera for other purpose it will fail